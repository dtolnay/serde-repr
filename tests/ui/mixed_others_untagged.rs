use serde_repr::Deserialize_repr;

#[derive(Deserialize_repr)]
#[repr(u8)]
enum MixedOthersUntagged {
    #[serde(untagged)]
    A,
    #[serde(other)]
    B,
}

fn main() {}
