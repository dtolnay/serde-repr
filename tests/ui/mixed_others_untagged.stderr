error: only one variant can be #[serde(other)] or #[serde(untagged)]
 --> tests/ui/mixed_others_untagged.rs:3:10
  |
3 | #[derive(Deserialize_repr)]
  |          ^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Deserialize_repr` (in Nightly builds, run with -Z macro-backtrace for more info)
